tslearn/docs/examples/plot_sdtw.py

```diff
@@ -62,7 +62,7 @@
 # fit this example)
 potential_matches = find_peaks(-cost_func, distance=sz * 0.75, height=-50)[0]
 
-# Calculate the optimal warping path starting from each of the identified minima
+# Calculate the optimal warping path starting from each of the minima
```

Comment:
I prefer the original formulation since `find_peaks` could have many different forms, depending on application.

Reply:
This is actually a change that I just did to make sure the PEP8 checks passed. I will revert it and split the comments over multiple lines or smth.